script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt

```diff
@@ -5,13 +5,13 @@ I spent a full two hours in the dining room.
 <Page2>
 "Welcome back, Shiki-sama. Although you did not instruct me to do so, I have prepared your bed."
 <Page3>
-...Or so she says.
+...she says.
```

Comment:
I feel like the change is more correct, but it just doesn't sound right for some reason

Reply:
It definitely reads kinda weird but I don't have an alternative right now.